### ISSUE TITLE
Fix packages page

### ISSUE THREAD

**Issue by tlienart** [JuliaLang/www.julialang.org#748]
This page: https://julialang.org/packages/ looks a bit weird.

1. ~~the h1 directly followed by h2 is pretty ugly~~
1. ~~it's missing some text explaining what the links point to~~

I'm happy to open a PR to make it nicer but would need some input from @StefanKarpinski and @logankilpatrick of what should ideally be there in terms of blurb + short description of the links 

PS: what points to that page?

**Comment by logankilpatrick** [comment]
I'll defer to @StefanKarpinski. I was just helping with the implementation itself, I am not sure what the motivation/vision is.

**Comment by ViralBShah** [comment]
I think `pkg.julialang.org` points here.

**Comment by StefanKarpinski** [comment]
The motivation is that https://pkg.julialang.org used be a doc search service, which was kind of weird because it was a Julia Computing proprietary service at a JuliaLang public URL, which was kind of questionable. Moreover, https://pkg.julialang.org is now a Pkg server and not a website. So we've made it redirect to https://julialang.org/packages/ as a page to list package discovery services such as JuliaHub and JuliaPackages, so the point of this page is just to list and describe those. I added the page hastily because the redirect has already been added and I didn't want it to just go to a 404 page. It should definitely be fleshed out with some text and explanation.
